web-common/src/layout/drag.ts

```diff
@@ -1,4 +1,15 @@
-export function drag(node, params) {
+import type { Writable } from "svelte/store";
+import type { LayoutElement } from "./workspace/types";
+
+interface DragParams {
+  store: Writable<LayoutElement>;
+  minSize?: number;
+  maxSize?: number;
+  reverse?: boolean;
+  orientation?: "horizontal" | "vertical";
+}
+
+export function drag(node, params: DragParams) {
```

Comment:
This was an issue before the PR, but the `mousemove` function needs to prevent the default browser behavior. Resizing selects elements in Safari.

Reply:
Nice catch, fixed 👍